modules/geo-layers/src/mvt-layer/coordinate-transform.js

```diff
@@ -0,0 +1,52 @@
+const availableTransformations = {
+  Point,
+  MultiPoint,
+  LineString,
+  MultiLineString,
+  Polygon,
+  MultiPolygon
+};
+
+function Point([pointX, pointY], {x, y, z}) {
+  const originX = x;
+  const originY = y;
+  const zoomLevelSize = Math.pow(2, z);
+
+  const y2 = 180 - ((pointY + originY) * 360) / zoomLevelSize;
+
+  return [
```

Comment:
Use `worldToLngLat` from `@math.gl/web-mercator`

Reply:
Used another approach getting the bounding box of the tile. With `viewport.projectFlat` and `viewport.unprojectFlat`